.github/workflows/ci.yml

```diff
@@ -51,7 +51,31 @@ jobs:
             nominatim &
           sleep 45
           ./assert-non-empty-json "http://localhost:8082/search.php?q=avenue%20pasteur"
+          
+      - name: Check import full style
+        working-directory: .github/workflows
+        run: |-
+          docker run -i --rm \
+            -e PBF_URL=http://download.geofabrik.de/europe/monaco-latest.osm.pbf \
+            -e REPLICATION_URL=http://download.geofabrik.de/europe/monaco-updates/ \
+            -e IMPORT_STYLE=full \
+            -p 8083:8080 \
+            nominatim &
+          sleep 45
+          ./assert-non-empty-json "http://localhost:8083/search.php?q=Paris"
 
+      - name: Check import admin style
+        working-directory: .github/workflows
+        run: |-
+          docker run -i --rm \
+            -e PBF_URL=http://download.geofabrik.de/europe/monaco-latest.osm.pbf \
+            -e REPLICATION_URL=http://download.geofabrik.de/europe/monaco-updates/ \
+            -e IMPORT_STYLE=admin \
+            -p 8085:8080 \
```

Comment:
we've skipped 8084, would change it to that one please! :)

Reply:
He did try that and something is already running on 8084, however I have no clue what it is.

https://github.com/mediagis/nominatim-docker/runs/2400837113?check_suite_focus=true#step:8:11

Reply:
Strange, thanks for the info.